var/spack/repos/builtin/packages/gearshifft/package.py

```diff
@@ -14,7 +14,9 @@ class Gearshifft(CMakePackage):
 
     maintainers = ['ax3l']
```

Comment:
```suggestion
    maintainers = ['zyzzyxdonta']
```

@h-denpo do you also want to co-maintain this package? :)

Reply:
I've already created a recipe for version 0.4.0. In that case, args.append('-DCMAKE_CXX_FLAGS=-pthread') was not necessary.

> @h-denpo do you also want to co-maintain this package? :)
No, sir. :-)  Because I'll be leaving soon.